.pfnci/schema.yaml

```diff
@@ -109,6 +109,14 @@ nccl:
       "11.4":
       "11.5":
       "11.6":
+  "2.12":
+    spec: "2.12.*"
+    cuda:
+      "10.2":
+      "11.0":
+      "11.4":
+      "11.5":
```

Comment:
```suggestion
```

The release notes says: This NCCL release supports [CUDA 10.2](https://docs.nvidia.com/cuda/archive/10.2/index.html), [CUDA 11.0](https://docs.nvidia.com/cuda/archive/11.0/index.html), and [CUDA 11.6](https://docs.nvidia.com/cuda/archive/11.6/index.html).

https://docs.nvidia.com/deeplearning/nccl/release-notes/rel_2-12-7.html#rel_2-12-7